platform.txt

```diff
@@ -79,7 +79,8 @@ compiler.elf2lst.extra_flags=
 
 # USB Flags
 # ---------
-build.usb_flags=-DUSBCON {build.usb_speed} -DUSBD_VID={build.vid} -DUSBD_PID={build.pid} -DHAL_PCD_MODULE_ENABLED
+#build.usb_flags=-DUSBCON {build.usb_speed} -DUSBD_VID={build.vid} -DUSBD_PID={build.pid} -DHAL_PCD_MODULE_ENABLED
+build.usb_flags=
```

Comment:
`build.usb_flags` is definned by menu selection, user can choose which usb controller they want to use as device: usbd, usbfs, usbhs